spot-client/src/spot-remote/ui/views/join-code-entry.js

```diff
@@ -65,15 +65,16 @@ export class JoinCodeEntry extends React.Component {
         this.props.dispatch(setJoinCode(''));
         remoteControlService.disconnect();
 
-        const queryParams = new URLSearchParams(this.props.location.search);
-        const code = queryParams.get('code');
+        const { pathname } = this.props.location;
+        const codeMatch = pathname.match(new RegExp('^/(\\w{6})$'));
+        const code = codeMatch && codeMatch[1];
```

Comment:
So the default already goes down to the remote control page?

Reply:
yes, here: https://github.com/jitsi/jitsi-meet-spot/blob/master/spot-client/src/app.js#L158

It doesn't have a path defined, that's why